management-api/pom.xml

```diff
@@ -9,28 +9,19 @@
     <version>1.0-SNAPSHOT</version>
 
     <parent>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.5.7.RELEASE</version>
+        <groupId>com.afh.skytouch</groupId>
+        <artifactId>skytouch-test</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
Here's where the maven property I talked about before will come in handy

Reply:
It didn´t allow me to add a proepertie to the spring boot parent